plugins/sdk.mjs

```diff
@@ -5,7 +5,7 @@ import { JsonRpcProvider } from "@ethersproject/providers";
 import { cache } from "./caching.mjs";
 import ms from "ms";
 
-const chains = [1, 250];
+const chains = [1, 250, 42161];
```

Comment:
Looks like there's an opportunity to be more DRY here with the same constants defined in `app.mjs`